prac_07/guitars.py

```diff
@@ -0,0 +1,44 @@
+from prac_07.guitar import Guitar
+
+
+def main():
+    # Create guitars list for storing
+    guitars = []
+
+    print("My Guitars!")
+    print("Enter blank name to show guitar list and exit!")
+    name = str(input("Name: "))
+    # Loop as long as user does not input empty name
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        guitar_info = Guitar(name, year, cost)
```

Comment:
I would use `guitar` rather than `guitar_info` which doesn't sound enough like it's a guitar. It implies to me it's in a different form/type. Something like `temp_guitar` would also be OK - see how that sounds like a Guitar?